cdk/config.py

```diff
@@ -64,14 +66,23 @@ class Deployment(BaseSettings):
         description="ARN of AWS Role used to validate access to S3 data"
     )
 
-    data_pipeline_arn: AwsStepArn = Field(
+    data_pipeline_arn: Optional[AwsStepArn] = Field(
         description="ARN of AWS step function used to trigger data ingestion"
     )
```

Comment:
should we just remove this since the endpoints have been changed to use airflow?
or we could have an option to run either